### DISCUSSION TITLE
Start of the DGD/Hydra Github forum

### DISCUSSION THREAD

**Discussion by dworkin** [dworkin/dgd#80] [Announcements]
The purpose of this forum is to discuss DGD, Hydra and related projects (lpc-doc and lpc-ext). It is **NOT** the place to report bugs; start an issue for those.

@shentino
@nyankers
@Miraculix
@nbmc
@diagonalfish
@noahgibbs
@silenus-dionysus
@sirdude
@zellski
@francipvb
@Blain20
@andreasfaerber
@seikichin
@neverbot
@november-kilo
@evilmog
@quixadhal
@bvl1999
@romland

**Comment by seikichin** [comment]
very Good



---Original---
From: "Felix A. ***@***.***&gt;
Date: Tue, Jan 3, 2023 21:32 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: [dworkin/dgd] Start of the DGD/Hydra Github forum (Discussion #80)




 
The purpose of this forum is to discuss DGD, Hydra and related projects (lpc-doc and lpc-ext). It is NOT the place to report bugs; start an issue for those.
 
@shentino
 @nyankers
 @Miraculix
 @nbmc
 @diagonalfish
 @noahgibbs
 @silenus-dionysus
 @sirdude
 @zellski
 @francipvb
 @Blain20
 @andreasfaerber
 @seikichin
 @neverbot
 @november-kilo
 @evilmog
 @quixadhal
